[automatic] Update 1 advisory for Registrator (#588)
mbauman/AdvisoryDB.jl: advisories/published/2025/JLSEC-2025-2.md

```toml
schema_version = "1.8.0"
id = "JLSEC-2025-2"
modified = 2026-07-18T02:39:19.201Z
published = 2025-10-08T17:41:37.190Z

> withdrawn = 2026-07-18T02:39:20.256Z

Reviewer: ```suggestion
```

# Command injection in `withpasswd()` function in Registrator.jl

> Registrator is a GitHub app that automates creation of registration pull requests for julia packages to the General registry. Prior to version 1.9.5, if the clone URL returned by GitHub is malicious (or can be injected using upstream vulnerabilities) a shell script injection can occur within the `withpasswd` function. Alternatively, an argument injection is possible in the `gettreesha `function. either of these can then lead to a potential RCE. Users should upgrade immediately to v1.9.5 to receive a fix. All prior versions are vulnerable. No known workarounds are available.

Reviewer: ```suggestion
### Impact

If the clone URL returned by GitHub is malicious (or can be injected using upstream vulnerabilities), a shell script injection can occur within the `withpasswd()` function.  This can then lead to a potential RCE.

### Patches

Users should upgrade immediately to v1.9.5. All prior versions are vulnerable.

### Workarounds

None

### References

Fixed by: https://github.com/JuliaRegistries/Registrator.jl/pull/448 (which is available in v1.9.5).

### Credits

Thanks to *splitline* from the DEVCORE Research Team for reporting this issue.```

### Credits

> Thanks to *splitline* from the DEVCORE Research Team for reporting this issue.```

Reviewer: ```suggestion
Thanks to *splitline* from the DEVCORE Research Team for reporting this issue.
```
